### PR TITLE
component table created

### PR DESCRIPTION
**Pull request by nagadarshan**
can you review it.

### CODE REVIEW

File: Lpicks-Core/src/app/core/table/table.component.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+})
+export class TableComponent implements OnInit {
+
+  @Input() tablecontent: TableContent;
+
+  constructor() { }
+
+  ngOnInit() {
+  }
+  arrayN(item: string[], n: number): string[] {
```
[the remark this answers is not in the document]
Comment (BSubashRaj) [reply Lpicks-Core/src/app/core/table/table.component.ts]: It is used to remove the extra fields if user accidentally adds more fields to a particular row  than the number of headers. suppose there are 3 columns and in JS object users sends 4 fields it just displays 3 fields and omits the 4th field.

File: Lpicks-Core/src/app/app.module.ts [hunk 2]
```diff
@@ -3,10 +3,14 @@ import { NgModule } from '@angular/core';
...
 import { AppRoutingModule } from './app-routing.module';
 import { AppComponent } from './app.component';
+import { TableComponent } from './core/table/table.component';
+import { DemoTableComponent } from './demo/demo-table/demo-table.component';
 
 @NgModule({
   declarations: [
-    AppComponent
+    AppComponent,
+    TableComponent,
```
[the remark this answers is not in the document]
Comment (nagadarshan) [reply Lpicks-Core/src/app/app.module.ts]: resolved it,I have added the component to core module

File: Lpicks-Core/src/app/core/table/table.d.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+interface TableContent {
+    header: [];
+    body: [][];
+}
+
+// interface BodyItem {
```
[the remark this answers is not in the document]
Comment (nagadarshan) [reply Lpicks-Core/src/app/core/table/table.d.ts]: removed the commented code

**Comment by Madhusuthanan-B** [comment]
Conflicts with master. Rebase please.

**Pull request merged**